tests/tests_pytorch/plugins/precision/test_deepspeed_precision.py

```diff
@@ -11,11 +11,31 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from unittest import mock
+
 import pytest
 
 from pytorch_lightning.plugins.precision.deepspeed import DeepSpeedPrecisionPlugin
+from pytorch_lightning.trainer.trainer import Trainer
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
 
 
 def test_invalid_precision_with_deepspeed_precision():
     with pytest.raises(ValueError, match="is not supported. `precision` must be one of"):
         DeepSpeedPrecisionPlugin(precision=64, amp_type="native")
+
+
+def test_deepspeed_precision_apex_not_installed(monkeypatch):
```

Comment:
I suggest to skip these tests on windows

Reply:
any reason?

https://github.com/NVIDIA/apex#experimental-windows

Reply:
Because in an earlier commit you had Trainer(strategy="deepspeed") and this fails on windows as deepspeed is not supported there.

Reply:
it's a unit test.. so won't matter.

Reply:
It does. `Trainer(strategy="deepspeed")` will raise an error if deepspeed is not installed